s3fs/core.py

```diff
@@ -448,6 +449,16 @@ def _open(
         kwargs: dict-like
             Additional parameters used for s3 methods.  Typically used for
             ServerSideEncryption.
+
+        Exceptions
+        ----------
+        FileExpired(OSError) :
```

Comment:
I'm not sure the long description is needed here since FileExpired has its own docstring and message

Reply:
I can move this description here to the docstring of the exception. This would make sense. In general it should be documented, that this error can happen in two circumstances:

1. On concurrent write/read accesses
2. When directory cache is outdated

Especially the second case is not straight forward an could lead to very long debugging sessions, if you don't know it.